File: plugin/AppsAction/src/main/java/AppsAction.java
import android.content.Context;
import android.graphics.drawable.Drawable;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;
import androidx.core.content.ContextCompat;
import com.aliucord.annotations.AliucordPlugin;
import com.aliucord.entities.Plugin;
import com.discord.databinding.WidgetChatListActionsBinding;
import com.discord.utilities.color.ColorCompat;
import com.lytefast.flexinput.R;

@AliucordPlugin
public class AppsAction extends Plugin {
    @Override
    public void start(Context ctx) {
        // Find the inner class for binding
        Class<?> bindingClass = null;
        for (Class<?> clazz : com.discord.widgets.chat.list.actions.WidgetChatListActions.class.getDeclaredClasses()) {
            if (clazz.getName().contains("$binding$")) {
                bindingClass = clazz;
                break;
            }
        }

        if (bindingClass != null) {
            patcher.after(bindingClass, "invoke", (callFrame) -> {
                WidgetChatListActionsBinding binding = (WidgetChatListActionsBinding) callFrame.getResult();
                Context context = binding.getRoot().getContext();
                ViewGroup layout = (ViewGroup) binding.a.getChildAt(0);

                TextView apps = new TextView(context, null, 0, R.i.UiKit_Settings_Item_Icon);
                apps.setText("Apps");
                apps.setVisibility(View.VISIBLE);

                Drawable drawable = ContextCompat.getDrawable(context, R.e.ic_authed_apps_24dp);
                if (drawable != null) {
                    drawable = drawable.mutate();
                    drawable.setTint(ColorCompat.getThemedColor(context, R.b.colorInteractiveNormal));
                }

                apps.setCompoundDrawables(drawable, null, null, null);
                layout.addView(apps);
            }, View.class);
        }
    }

    @Override
    public void stop(Context context) {
        patcher.unpatchAll();
    }
}

File: AppsAction/src/main/kotlin/AppsAction.kt
import android.content.Context
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.core.content.ContextCompat
import com.aliucord.annotations.AliucordPlugin
import com.aliucord.entities.Plugin
import com.aliucord.patcher.after
import com.discord.databinding.WidgetChatListActionsBinding
import com.discord.utilities.color.ColorCompat
import com.discord.widgets.chat.list.actions.`WidgetChatListActions$binding$2`
import com.lytefast.flexinput.R
import kotlin.properties.Delegates

@AliucordPlugin
class AppsAction : Plugin() {
    private var WidgetChatListActionsBinding.apps by Delegates.notNull<TextView>()

    override fun start(ctx: Context) {
        patcher.after<`WidgetChatListActions$binding$2`>("invoke", View::class.java) {
            val binding = it.result as WidgetChatListActionsBinding
            val context = binding.root.context
            val layout = binding.a.getChildAt(0) as ViewGroup

            binding.apps = TextView(context, null, 0, R.i.UiKit_Settings_Item_Icon).apply {
                @Suppress("SetTextI18n")
                text = "Apps"
                visibility = View.VISIBLE

                val drawable = ContextCompat.getDrawable(context, R.e.ic_authed_apps_24dp)!!
                    .mutate()
                    .apply {
                        setTint(ColorCompat.getThemedColor(context, R.b.colorInteractiveNormal))
                    }

                setCompoundDrawables(drawable, null, null, null)
            }
            layout.addView(binding.apps)
        }
    }

    override fun stop(context: Context) = patcher.unpatchAll()
}